documentation.md

```diff
@@ -44,7 +44,7 @@ per user.followers()
 - [Flyrobu](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#flyrobu)
 - [HealthGrades](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#healthgrades)
 - [IMDB](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#imdb)
-
+- [Wuzzuf](https://github.com/Clueless-Community/scrape-up/blob/main/documentation.md#Wuzzuf)
```

Comment:
In the `documentation.md` file, we maintain the documentation of the modules that are in the deployed version of the package only. 

Remove it from here and include it in `dev-documentation.md` file. Also do not just add link to the location. Also document you modules like this
```js
## Dictionary.com

First create an object of class `Dictionary`.


| Methods          | Details                                                                                      |
| ---------------- | -------------------------------------------------------------------------------------------- |
| `.get_word_of_the_day()`       | Returns the word of the day. |
| `.word_of_the_day_definition()`    | Returns the definition of the word of the day.
--------

```